test/utils/update_resources.go

```diff
@@ -28,11 +28,12 @@ import (
 
 const (
 	// Parameters for retrying updates/waits with linear backoff.
-	// TODO: Try to move this to exponential backoff by modifying scale.Scale().
+	// TODO -> REVIEW: Try to move this to exponential backoff by modifying scale.Scale().
 	updateRetryInterval = 5 * time.Second
 	updateRetryTimeout  = 1 * time.Minute
 	waitRetryInterval   = 5 * time.Second
 	waitRetryTimeout    = 5 * time.Minute
+	retryAmount         = 12
```

Comment:
Can you explain why the retry number is set to 12? Just arbitrary or there is a meaning behind this?

Reply:
It's is random one now sure how many time to retry